ethereum.md

```diff
@@ -764,5 +663,188 @@ TODO: case with nonzero ommers.
     rule check TESTID : { "uncleHeaders" : OMMERS } => check "ommerHeaders" : OMMERS ~> failure TESTID
  // --------------------------------------------------------------------------------------------------
     rule <k> check "ommerHeaders" : [ .JSONList ] => . ... </k> <ommerBlockHeaders> [ .JSONList ] </ommerBlockHeaders>
+```
```

Comment:
Can we break this out into a separate file to emphasize that most of the semantics doesn't depend on the binary encoding?